roslibrust/src/comm.rs

```diff
@@ -76,7 +80,7 @@ impl FromStr for Ops {
 
 /// Describes the low level comm capabilities of talking to a rosbridge server
 #[async_trait]
-pub trait RosBridgeComm {
+pub(crate) trait RosBridgeComm {
```

Comment:
Is the intention to not allow external code to impl this trait?

Reply:
It is "for now" I don't think anyone should be impl-ing until we have a better plan going forward.